ui-v2/app/services/client/http.js

```diff
@@ -63,6 +63,9 @@ export default Service.extend({
       });
     }
   },
+  abort: function(id = null) {
+    get(this, 'connections').purge();
+  },
```

Comment:
We've included an id argument here, to remind us at some point that it maybe useful to be able to `abort` individual connections here. We initially named it `abortAll`, and then changed to just `abort` when we figured this might be useful.